### PR TITLE
Filling in documentation

### PR DESCRIPTION
**Pull request by Mwthorn**
Getting more used to git. Unfortunately i lost a few commits during this.

### CODE REVIEW

File: plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 1]
```diff
@@ -1464,6 +1464,9 @@ public void registerCoreMembers() {
         // @Description
         // The flag command sets or modifies custom value storage database entries connected to
         // each player, each NPC, each entity, and the server.
+        // All the flag values are stored default in "plugins/denizen/saves.yml" file.
+        // For an alternative way of storing values, use either yaml (See <a href="/cmds/yaml">command/YAML</a>)
+        // or sql (See <a href="/cmds/SQL">command/SQL</a>)
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: This is not how links work in dMeta.

File: plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 2]
```diff
@@ -2429,7 +2432,12 @@ public void registerCoreMembers() {
         // - modifyblock cu@<player.location>|<player.location.cursor_on> li@stone|dirt li@25|25
         //
         // @Usage
-        // TODO: Document Command Details
+        // Use to modify an area around naturally.
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: You gotta work on English sentences yo

File: plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java (hunk 1 above)
Comment (Mwthorn) [inline plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: Could you give an example of a link? Then I'll fix it in various other places as well.

**Comment by Mwthorn** [comment]
Updated commits. Sorry for the delay.

**Pull request merged**